src/tests/functional/navigation_tests.ts

```diff
@@ -72,6 +72,7 @@ export class NavigationTests extends TurboDriveTestCase {
 
   async "test following a same-origin [target] link"() {
     this.clickSelector("#same-origin-targeted-link")
+    await this.nextBeat
```

Comment:
Could we omit this is we moved the `await` to the `this.clickSelector` call?

Reply:
This "fixed" [a failure in Firefox on CI](https://github.com/hotwired/turbo/runs/1626385513?check_suite_focus=true#step:7:83) that I couldn't reproduce locally. I don't think we need to add a delay to every click, and I suspect it has more to do with switching window handles https://github.com/hotwired/turbo/blob/cd09a3be3c85d9cdaae20954193cd0e4f75320e9/src/tests/functional/navigation_tests.ts#L76